connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/ClusterConfigState.java

```diff
@@ -112,12 +112,13 @@ public boolean contains(String connector) {
      * {@link org.apache.kafka.common.config.ConfigTransformer} by having all variable
      * references replaced with the current values from external instances of
      * {@link ConfigProvider}, and may include secrets.
+     * NOTED: If the configuration is not null, it will have been transformed.
```

Comment:
Nit:  Sorry, I wasn't that a clear.  I meant to remove this NOTE and start the JavaDoc with:

```
/**
     * Get the configuration for a connector.  If the configuration is not null,
     * it will have been transformed by
```